resources/views/store/manage_vouchers.blade.php

```diff
@@ -32,11 +32,10 @@
                     <div class="icon-container warning">
                         <i class="fa fa-exclamation-triangle" aria-hidden="true"></i>
                     </div>
-                    <div>
+                    <div id="family-warning">
                         @foreach( $noticeReasons as $notices )
                             <p class="v-spaced">
-                                Warning: {{ $notices['count'] }} {{ str_plural($notices['entity'], $notices['count']) }}
-                                is {{ $notices['reason'] }}
+                                Warning: {{ $notices['count'] }} {{ str_plural($notices['entity'], $notices['count']) }} currently "{{ $notices['reason'] }}"
```

Comment:
Is this the bit where it should be `Reminder`? if so, maybe update the id as well.

Reply:
This will change in the card Kat and Faith are looking at.